### DISCUSSION TITLE
Windows Community Toolkit - how complete is the implementation?

### DISCUSSION THREAD

**Discussion by iot-crazy** [unoplatform/uno#8529] [Q&A] [answered]
Hi All,

I'm considering using the windows community toolkit with an Uno project and dotnet 6. Please can you advise how complete the implementation is of this?

regards
Tony

**Comment by jeromelaban** [comment, marked as answer]
.NET 6 support was recently added to the WinUI targets (The `Uno.CommunityToolkit.*` nuget packages). The implementation is basically the same as the one from UWP, and uses the same source as the official repository.

Now uno itself may not have all the features that the toolkit controls and features need, and you'll need to let us know when you find those. As a rule of thumb, we try to keep the most frequently used controls (i.e. DataGrid, WrapPane, etc...) up to date and in as much lock step as possible with the toolkit team. 

Is there a specific control or feature that you're interested in? It is important to check in often as this toolkit story is a wide topic and it always evolve with more controls, and control properties, being covered as time goes on.

**Reply by iot-crazy** [reply]
For now we're just looking at what tools are available for the next phase of a POC (basically taking a small project to production to see how it fares when we realase it into the wild as an internal app) and to help make a decision on a larger project.
